src/EA.Weee.RequestHandlers/AatfReturn/Internal/AatfDataAccess.cs

```diff
@@ -216,5 +216,27 @@ public async Task RemoveAatfData(Aatf aatf, IEnumerable<int> quarters)
         {
             return w => w.AatfId == aatf.Id && w.Return.Quarter.Year == aatf.ComplianceYear && (int)w.Return.Quarter.Q == quarter;
         }
+
+        public async Task<List<short>> GetComplianceYearsForAatfByAatfId(Guid aatfId)
+        {
+            return await context.Aatfs
+                    .Where(r => r.AatfId == aatfId)
+                    .Select(r => r.ComplianceYear)
+                    .Distinct()
+                    .OrderByDescending(year => year)
+                    .ToListAsync();
+        }
+
+        public async Task<Guid> GetAatfId(Guid aatfId, short complianceYear)
```

Comment:
Rename this to GetAatfByAatfIdAndComplianceYear

Reply:
Renamed